src/vulnpy/trigger/__init__.py

```diff
@@ -1,3 +1,14 @@
 from vulnpy.trigger.util import create_trigger_map, get_trigger  # noqa: F401
 
 TRIGGER_MAP = create_trigger_map()
```

Comment:
hmm wondering if at this point we should memoize this? 🤔

Reply:
I stuck a debugger inside `create_trigger_map` to see if it gets called multiple times and it doesn't